solidity/dashboard/package.json

```diff
@@ -16,7 +16,7 @@
     "formik": "^2.1.3",
     "moment": "^2.20.1",
     "web3": "^1.2.4",
-    "@keep-network/keep-core": ">0.14.0-pre <0.14.0-rc"
+    "@keep-network/keep-core": "0.13.0-rc"
```

Comment:
14.

Reply:
Oh, wait, no.

These should always increment in ranges.`>0.14.0-rc < 0.14.0`

Reply:
Oh my. Why... why... 😭